### PR TITLE
s390x_cpu_model: Add baseline test cases

### PR DESCRIPTION
**Pull request by smitterl**
The `query-cpu-model-baseline` is expected select
- the intersection of both model definitions' properties
- the older generation in a list of two.

Agreed with developers not to cover all cpu models.

Signed-off-by: Sebastian Mitterle <smitterl@redhat.com>
Suggested-by: Boqiao Fu <bfu@redhat.com>

### CODE REVIEW

**Comment by smitterl** [comment]
```
JOB ID     : 9efd0740673f790e8730ecce4fcf0bef4de6b9bc
JOB LOG    : /home/kar/workspace/job-results/job-2022-08-11T04.45-9efd074/job.log
 (1/1) type_specific.io-github-autotest-qemu.s390x_cpu_model.cpu_baseline: STARTED
 (1/1) type_specific.io-github-autotest-qemu.s390x_cpu_model.cpu_baseline: PASS (18.38 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/kar/workspace/job-results/job-2022-08-11T04.45-9efd074/results.html
JOB TIME   : 19.64 s
```

**Comment by smitterl** [comment]
@fbq815 Please can you help review

**Comment by fbq815** [comment]
LGTM, @PaulYuuu could you please help review and merge this patch?

**Comment by PaulYuuu** [comment]
Code almost good to me, but please fix the CI style check first.

**Comment by smitterl** [comment]
@PaulYuuu @fbq815 I now fixed the style issues but also need maintainer approval for first-time contributors; please, can you help?

**Review by fbq815 — APPROVED** [review]

**Review by PaulYuuu — APPROVED** [review]

**Comment by smitterl** [comment]
Re-run after style fix:
```
# avocado run --vt-type qemu --test-runner=nrunner --vt-machine-type s390-virtio s390x_cpu_model.cpu_baseline --vt-extra-params 'password = redhat'
JOB ID     : 144d7e4591dc943fe96746ab97e1847a2479ac82
JOB LOG    : /home/kar/workspace/job-results/job-2022-08-25T05.02-144d7e4/job.log
 (1/1) type_specific.io-github-autotest-qemu.s390x_cpu_model.cpu_baseline: STARTED
 (1/1) type_specific.io-github-autotest-qemu.s390x_cpu_model.cpu_baseline: PASS (23.20 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/kar/workspace/job-results/job-2022-08-25T05.02-144d7e4/results.html
JOB TIME   : 24.56 s
```

**Pull request merged**
